Mathlib/CategoryTheory/Equivalence.lean

```diff
@@ -62,32 +62,39 @@ open CategoryTheory.Functor NatIso Category
 -- declare the `v`'s first; see `CategoryTheory.Category` for an explanation
 universe v₁ v₂ v₃ u₁ u₂ u₃
 
-/-- We define an equivalence as a (half)-adjoint equivalence, a pair of functors with
-  a unit and counit which are natural isomorphisms and the triangle law `Fη ≫ εF = 1`, or in other
-  words the composite `F ⟶ FGF ⟶ F` is the identity.
+/-- An equivalence of categories.
 
-  In `unit_inverse_comp`, we show that this is actually an adjoint equivalence, i.e., that the
-  composite `G ⟶ GFG ⟶ G` is also the identity.
+We define an equivalence between `C` and `D` as a half-adjoint equivalence: a pair of functors
+`F : C ⥤ D` and `G : D ⥤ C` with a unit `η : 𝟭 C ≅ F ⋙ G` and counit `ε : G ⋙ F ≅ 𝟭 D`, such that
+the natural isomorphisms `η` and `ε` satisfy the triangle law for `F`: namely, `Fη ≫ εF = 𝟙 F`. Or,
+in other words, the composite `F ⟶ FGF ⟶ F` is the identity.
```

Comment:
nit: consider
```suggestion
in other words, the composite $F ⟶ FGF ⟶ F$ is the identity.
```
since this isn't valid Lean code.

Reply:
Hmm, good point, but then we'd have to change `⟶` to `\to` or equivalent for LaTeX, and it would be less legible in hovers since we don't have LaTeX rendering there.

What do you think of "Or, more informally," instead of "Or, in other words" to try to get the best of both worlds?

Reply:
I think the latex renderer is pretty happy to take random unicode arrows, so it might work fine as I write it above.

This isn't a hard rule; but I think it's handy to know from formatting alone what is lean code and what is not. If you disagree then fine to leave as you have it, I don't really care what the english around it introduces it with.